plugins/processors/reversedns/parallel/unordered.go

```diff
@@ -31,18 +31,18 @@ func NewUnordered(acc telegraf.MetricStreamAccumulator, workerCount int64) *Unor
 	return p
 }
 
-func (p *Unordered) Do(fn func(acc telegraf.MetricStreamAccumulator)) {
+func (p *Unordered) Do(fn func(acc telegraf.Accumulator)) {
 	p.sem.Acquire(context.TODO(), 1)
 	go func() {
-		fn(p)
+		fn(p.acc)
```

Comment:
This is a breaking change. The previous version used the idea that Unordered matched the accumulator interface so it could write to queue.